### DISCUSSION TITLE
Federating Storybooks?

### DISCUSSION THREAD

**Discussion by danielsdesk** [module-federation/module-federation-examples#629] [Q&A] [answered]
Hi! Last year the way to do federation of Storybooks was to use this: https://github.com/xolvio/storybook-webpack-federation-plugin
I'm curious to know if this is still the way we should federate storybooks or if there is a better way to do it now, considering all the advancements that have happened (e.g. federating next.js)? It appears that Storybook is still trying to get to Webpack 5? From taking a look at what's going on over there it seems they are not that far from it (at least updating their build steps to webpack 5, not counting addons), though I thought for good measure I'd ask directly in the MF community to check. Thanks!

**Comment by jacob-ebey** [comment, marked as answer]
I've given up exploring until webpack 5 works out of the box with storybook. They have soo many stages in their process where they modify the config it feels impossible to get it to work with the hooks we are given as a user.

**Reply by danielsdesk** [reply]
Got it; I figured. For now I will look at that other webpack 4 plugin and mark this as answered. Whenever they get up to full webpack 5 compatibility I'll revisit asking this. Thanks!

**Reply by jacob-ebey** [reply]
@danielsdesk Revisiting it seems Storybook as dropped beta / prerelease support for webpack 5.
